content/projects/kauri-staking/index.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: "NavPi 2.0 'Kowhai'"
+title: "Kauri Staking"
 date: 2018-02-20T22:04:39+13:00
 lastmod: 2018-02-20T22:04:39+13:00
 draft: false
-author: "Mike Delucchi"
-description: "NavPi Kowhai is a complete rebuild of the NavPi StakeBox software with improved performance, security and usability."
+author: "Paul Sanderson"
+description: "Enabling the Kauri Wallet to operate as a staking node on your desktop, remotely on a raspberry pi or even using thin staking on your mobile device."
```

Comment:
I'm not entirely sure we can claim it will run on a Raspberry Pi tbh, with the issues we've had with resources on the NavPi. Just being devil's advocate.

Reply:
it will be fine. Go is much more lightweight and we can optimize the Raspberry build.

Reply:
👍